packages/react-link/src/common/isConformant.ts

```diff
@@ -10,6 +10,7 @@ export function isConformant<TProps = {}>(
     componentPath: module!.parent!.filename.replace('.test', ''),
     disabledTests: [`has-docblock`],
     extraTests: makeStylesTests as TestObject<TProps>,
+    skipAsPropTests: true,
```

Comment:
nit: add this link as comment: https://github.com/microsoft/fluentui/issues/19522

Reply:
Added!